keel-sql/src/main/resources/db/changelog/20200109-create-task-tracking-table.yml

```diff
@@ -0,0 +1,43 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-task-tracking-table
+      author: gyardeni
+      changes:
+        - createTable:
+            tableName: task_tracking
+            columns:
+              - column:
+                  name: task_id
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: task_name
+                  type: longtext
+                  value: "{}"
+                  constraints:
+                    nullable: false
+              - column:
+                  name: resource_id
```

Comment:
Not sure if `resource_id` should be a part of this table, we have to environment constraints that launch orca tasks so they aren't necessarily tied to a single resource.

Reply:
Maybe a generic field like `source` that we form with a prefix like `resource:` or `environment:` plus the identifier? I'm guessing this is mostly for logging/debugging purposes since the important bit is really just the `task_id`?

Reply:
Thanks for the feedback - to keep this data as generic as possible for future uses as well (we might want to include errors from ocra for constraints as well), will rename the `resource_id` to `subject`, like the actual field in the taskLauncher.